src/tainting/Taint.ml

```diff
@@ -130,6 +130,16 @@ and compare_sources s1 s2 =
       Option.compare compare_precondition s1.precondition s2.precondition
   | other -> other
 
+and compare_args a1 a2 =
+  let pos1 = a1.pos in
+  let pos2 = a2.pos in
+  match Stdlib.compare pos1 pos2 with
```

Comment:
I wonder if we should avoid `Stdlib.compare` here too!

Reply:
I'll open a separate PR to experiment with options